pkg/option/config.go

```diff
@@ -2591,13 +2598,37 @@ func (c *DaemonConfig) validateIPv6ClusterAllocCIDR() error {
 	return nil
 }
 
+func (c *DaemonConfig) validateIPv6NAT46x64CIDR() error {
+	ip, cidr, err := net.ParseCIDR(c.IPv6NAT46x64CIDR)
+	if err != nil {
+		return err
+	}
+
+	if cidr == nil {
```

Comment:
This cannot be true if `err == nil` and can also be removed in `validateIPv6ClusterAllocCIDR`.